### ISSUE TITLE
[Accessibility][Bug] - Wizard component - In the widget tab, every tab is marked as a Navigation Region

### ISSUE THREAD

**Issue by MonishaK28** [SAP/fundamental-ngx#13561]
### Describe your proposal

In the wizard component - In the Widget tab - each individual tab is marked as a navigation region (using role="navigation" or similar). This leads screen readers to treat every tab as a separate navigation section, which is misleading and cluttered, especially for users relying on assistive technologies.  

Only actual navigation areas (like menus or global navigation) should use the role=navigation. Tabs should be marked using appropriate roles, such as role=tab within a role=tablist container, to ensure accurate and meaningful structure for screen reader users. 

<img width="3657" height="1900" alt="Image" src="https://github.com/user-attachments/assets/72560f33-99eb-477e-afbe-74866cb51441" />

### Which versions of Angular and Fundamental Library for Angular are affected?

Fundamental Library - v0.56.04

### Declaration

- [x] I'm not disclosing any internal or sensitive information.
